crt_portal/features/templatetags/feature_script.py

```diff
@@ -23,7 +23,7 @@ def feature_script():
     ], separators=(',', ':'))
 
     script = textwrap.dedent(f"""
-        <script>
+        <script nonce="{{ request.csp_nonce }}">
```

Comment:
just noting after talking that this needs to be fixed, so I can be notified when you re-request review 😅

Reply:
updated! it looks like you don't actually see the csp_nonce locally though (I checked some existing instances) so you just see <script nonce>